FinalProject/fb_client_secrets.json

```diff
@@ -1,6 +1,6 @@
 {
   "web": {
-    "app_id": "YOUR_APP_ID_GOES_HERE",
-    "app_secret": "YOUR_APP_SECRET_GOES_HERE"
+    "app_id": "438142313021501",
+    "app_secret": "b3b7a4ca63439ead84bca9b04a791920"
```

Comment:
@jon-engelbert  Never commit your `client_secrets.json`. Also, can you break your pull requests into smaller one because the amount of changes is overwhelming for a single pull request. Break it into smaller one and I will review and merge.